### PR TITLE
Added whois and email functionality

### PR DESCRIPTION
**Pull request by ryansmick**
Implemented a Django app to determine the owner of a website and
send that owner an email

### CODE REVIEW

File: edge_cms_mail/owner_conf/views.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	owner = siteInfo.name
+	email = siteInfo.email
+	try:
+		send_mail(
+			"Please Confirm Email Address",
+			'''Hello {},
+				Thank you for using edge-cms on your domain: {}'''.format(owner, domain),
+			"DoNotReply@edge-cms.com",
+			[email],
+			fail_silently=True)
```
Comment (dmattia) [inline edge_cms_mail/owner_conf/views.py]: Could you expand on why you have fail_silently = True as well as a check for smtplib.SMTPException?

File: edge_cms_mail/edge_cms_mail/settings.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
```
Comment (dmattia) [inline edge_cms_mail/edge_cms_mail/settings.py:41]: can you add owner_conf to installed_apps

File: edge_cms_mail/edge_cms_mail/settings.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '$$35*a6fj60t!1_n7#1qrvw^z39e*8iaf9$k&i#*v=^8f!6e(i'
```
Comment (dmattia) [inline edge_cms_mail/edge_cms_mail/settings.py]: Use an environment variable if possible, or generate a "testing" key, or add the settings file to the .gitignore

File: edge_cms_mail/manage.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import os
```
Comment (dmattia) [inline edge_cms_mail/manage.py:2]: Would be nice to have a .gitignore file with *.pyc to remove compiled python

File: edge_cms_mail/owner_conf/views.py (hunk 1 above)
Comment (ryansmick) [reply edge_cms_mail/owner_conf/views.py]: Whoops, I meant to have it false. I'll make all of the changes and resubmit.

File: edge_cms_mail/owner_conf/views.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+		return JsonResponse({"status": "Failure", "message": "Unable to find owner for {}".format(domain)})
+
+	# Obtain owner info and send email
+	owner = siteInfo.name
+	email = siteInfo.email
+	try:
+		send_mail(
+			"Please Confirm Email Address",
+			'''Hello {},
+				Thank you for using edge-cms on your domain: {}'''.format(owner, domain),
```
Comment (dmattia) [inline edge_cms_mail/owner_conf/views.py]: possible extra tab

**Comment by ryansmick** [comment]
I added the new changes in the "Made Appropriate Changes" commit, which should be included in this pull request.

**Pull request merged**